tests/apis/test_apple_operational_info.py

```diff
@@ -0,0 +1,167 @@
+#   Copyright (C) 2020 Presidenza del Consiglio dei Ministri.
+#   Please refer to the AUTHORS file for more information.
+#   This program is free software: you can redistribute it and/or modify
+#   it under the terms of the GNU Affero General Public License as
+#   published by the Free Software Foundation, either version 3 of the
+#   License, or (at your option) any later version.
+#   This program is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+#   GNU Affero General Public License for more details.
+#   You should have received a copy of the GNU Affero General Public License
+#   along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+import json
+from datetime import date
+from http import HTTPStatus
+from typing import Any, Dict
+from unittest.mock import MagicMock, patch
+
+from pytest import fixture, mark
+from pytest_sanic.utils import TestClient
+
+from immuni_analytics.core import config
+from immuni_analytics.core.managers import managers
+from immuni_analytics.helpers.redis import get_upload_authorization_member_for_current_month
+from immuni_analytics.models.operational_info import OperationalInfo
+from immuni_common.models.enums import Platform
+from tests.fixtures.operational_info import OPERATIONAL_INFO
+
+ANALYTICS_TOKEN = (
+    "0a17753fecc38a5e259319e4524b55df439a98c1ff6326df7247263aa1192701cbe8799457cb1ac173590"
+    "eecb11bfe62a34cc0798f95d8842124814c24f53ff"
+)
+
+
+@fixture
+def headers() -> Dict[str, str]:
+    return {
+        "Authorization": f"Bearer {ANALYTICS_TOKEN}",
+        "Immuni-Dummy-Data": "0",
+        "Content-Type": "application/json; charset=utf-8",
+    }
+
+
+@patch("immuni_analytics.helpers.redis._LOGGER.info")
+async def test_apple_operational_info_with_exposure(
+    redis_logger_info: MagicMock,
```

Comment:
minor: OCD, please order them alphabetically

Reply:
Cannot as the patched one always comes first.